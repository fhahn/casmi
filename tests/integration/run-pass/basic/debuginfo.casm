// cmdline "-dall"

rule main = {
  debuginfo trace "Das ist ein Test: " + 2
  debuginfo a "a"
  debuginfo b "b"
  debuginfo c "c"
  program( self ) := undef
}
init main
