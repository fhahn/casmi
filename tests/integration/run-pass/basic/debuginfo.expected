trace: Das ist ein Test: 2
a: a
b: b
c: c
1 step later...
